src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -84,25 +84,37 @@ internal static class TestCommandParser
         public static readonly Option<bool> BlameCrashOption = new ForwardedOption<bool>("--blame-crash", LocalizableStrings.CmdBlameCrashDescription)
             .ForwardAs("-property:VSTestBlameCrash=true");
 
-        public static readonly Option<string> BlameCrashDumpOption = new ForwardedOption<string>("--blame-crash-dump-type", LocalizableStrings.CmdBlameCrashDumpTypeDescription)
+        public static readonly Option<string> BlameCrashDumpOption = CreateBlameCrashDumpOption();
+
+        private static Option<string> CreateBlameCrashDumpOption()
         {
-            ArgumentHelpName = LocalizableStrings.CrashDumpTypeArgumentName,
+            Option<string> result = new ForwardedOption<string>("--blame-crash-dump-type", LocalizableStrings.CmdBlameCrashDumpTypeDescription)
+            {
+                ArgumentHelpName = LocalizableStrings.CrashDumpTypeArgumentName,
+            }
+            .ForwardAsMany(o => new[] { "-property:VSTestBlameCrash=true", $"-property:VSTestBlameCrashDumpType={o}" });
+            result.AcceptOnlyFromAmong(new string[] { "full", "mini" });
```

Comment:
this method is no longer an extension method that returns the option (BCL requirement is that only builder types are allowed to do this)